docs/_config.yml

```diff
@@ -5,14 +5,12 @@
 # in the templates via {{ site.myvariable }}.
 title: gRPC-Gateway
 description: gRPC-Gateway Documentation Website
-# baseurl: "/grpc-gateway" # the subpath of your site, e.g. /blog
-# url: "https://grpc-gateway.github.io" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
Why did you remove this? Shouldn't this be set? https://stackoverflow.com/questions/50775454/what-should-the-baseurl-url-of-the-jekyll-blog

Reply:
@johanbrandhorst Reason for removing this is that since the GitHub Pages site is currently being built from the /docs folder in the master branch so I thought we do not need `baseurl` but I think we should revert the changes.